### ISSUE TITLE
`docker.Secret` docs page is missing `Example Usage` section

### ISSUE THREAD

**Issue by andrzejressel** [pulumi/pulumi-docker#1294]
### Describe what happened

Pulumi docs: https://www.pulumi.com/registry/packages/docker/api-docs/secret/ is missing whole section `Example Usage` that is available in terraform docs: https://registry.terraform.io/providers/kreuzwerker/docker/latest/docs/resources/secret

### Sample program

N/A

### Output of `pulumi about`

N/A

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by mjeffryes** [comment]
@blampe I don't think this provider is under ci mgmt (probably due to being a hybrid bridged/native provider it didn't quite fit before?) If we can get it under ci-mgmt, it's just a flag to enable the generated frontmatter docs. 

However, this is a resource doc, so it's not really about those recent frontmatter changes. I wonder if this repo is also behind on picking up latest tfgen changes, especially the newer example converter.

**Comment by blampe** [comment]
@mjeffryes it is under ci-mgmt but it has a custom [docsCmd](https://github.com/pulumi/pulumi-docker/blob/master/.ci-mgmt.yaml#L25). I wonder if that's the source of these problems, or if it's even necessary anymore. 

> I wonder if this repo is also behind on picking up latest tfgen changes, especially the newer example converter.

Almost certainly. I'm not sure what's involved in picking that up, but if someone can point me in the right direction I can take a look.
